### ISSUE TITLE
passenger_app_log_file doesn't log all errors into app log

### ISSUE THREAD

**Issue by avkarenow** [phusion/passenger#2106]
Hello,

I'm using Passenger 5.3.4 on nginx.
Unfortunately, Passenger doesn't log all logs into app (separate) log when passenger_app_log_file is used.
There is an example application logged into the main log:
> app = proc do |env|
> [200, { "Content-Type" => "text/html" }, ["Ruby"]
> end
> ruun app

(ruun instead run)

And there application which is logged correctly to its file:
> app = proc do |env|
> [200, { "Content-Type" => "text/html" }, ["Ruby"]
> logmenow
> end
> run app
(logmenow is undefined and its logged into file from passenger_app_log_file)

Any chances to fix that?

**Comment by CamJN** [comment]
Does the missing line show up in the nginx error log? Or is it missing from both?

**Comment by avkarenow** [comment]
Yes, it logs into nginx error log correctly, only app log is empty in this case.
